e2e/cypress/integration/interactive_menu/basic_options_spec.js

```diff
@@ -191,20 +195,23 @@ describe('Interactive Menu', () => {
     });
 
     it('IM21042 - "No items match" feedback', () => {
-        const missingUser = Date.now();
+        const missingUser = '1577793817254';
         const userOptions = getMessageMenusPayload({dataSource: 'users'});
 
         // # Post an incoming webhook for interactive menu with user options
         cy.postIncomingWebhook({url: incomingWebhook.url, data: userOptions});
 
         // # Get message attachment from the last post
         cy.getLastPostId().then((postId) => {
-            cy.get(`#messageAttachmentList_${postId}`).as('messageAttachmentList');
-        });
+            cy.get(`#messageAttachmentList_${postId}`).within(() => {
+                // # Type the missing user in the select input
+                cy.findByPlaceholderText('Select an option...').click().clear().type(`${missingUser}`);
 
-        cy.get('@messageAttachmentList').within(() => {
-            cy.get('.select-suggestion-container > input').click().clear().type(`${missingUser}`);
-            cy.get('.suggestion-list__no-results').should('be.visible').should('have.text', `No items match ${missingUser}`);
+                cy.get('#suggestionList').within(() => {
+                    // * Check if we get appropriate message when no options matches entered text
+                    cy.findByText(/No items match/gmui).should('exist');
```

Comment:
Could you please update to meet the test expectation of "No items match [what you have typed]" is displayed above the input/selection box"?

Reply:
Yes so if i write in template literals ``No item match ${selectes}``, tests are unable to find it. So i had to remove it. How do i write the variable in regex ?

Reply:
Anyways i reverted the change